static/js/settings_account.js

```diff
@@ -70,7 +70,11 @@ exports.append_custom_profile_fields = function (element_id, user_id) {
     all_custom_fields.forEach(function (field) {
         var field_type = field.type;
         var type;
-        var value = people.get_custom_profile_data(user_id, field.id);
+        var values = people.get_custom_profile_data(user_id, field.id);
+        if (values === undefined || values === null) {
+            values = {value:"", rendered_value:""};
```

Comment:
There should be a space after the `:`s here.

Reply:
I fixed our linter to catch this category of issue, see: bdb3da45044cdef93b523692e475f3644631ddb7

Reply:
Will fix.